entity-framework/core/modeling/data-seeding.md

```diff
@@ -88,4 +88,4 @@ Depending on the constraints of your deployment the initialization code can be e
 * Running the initialization app locally
 * Deploying the initialization app with the main app, invoking the initialization routine and disabling or removing the initialization app.
 
-This can usually be automated by using [publish profiles](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/visual-studio-publish-profiles).
```

Comment:
Should we create AKA links for these?

Reply:
@ajcvickers - Is there specific reason we are creating AKA links? Content on docs site should be referenced through relative URLs (everything lives in docs.microsoft.com domain). The destination repos will take care of redirects and locales.

Reply:
@smitpatel Probably just me getting it wrong.